### PR TITLE
[luci] Set quantization type of outputs

### PR DESCRIPTION
**Pull request by jinevening**
This sets quantization type of outputs

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
@jinevening , can you please explain purpose of this change? I can guess but I'm not sure.

**Comment by jinevening** [comment]
@seanshpark Currently, the quantization pass only quantizes values in circle nodes. This raises an assert (the output type of loco graph should be the same with the type of CircleOutput) when circle2circle calls `validate`.

This PR sets the quantization type to the output of loco graph (if `from` of CircleOutput is quantized) to avoid the assertion.

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -581,6 +581,18 @@ bool QuantizeWithMinMaxPass::run(loco::Graph *g)
     circle_node->accept(&qb);
   }
 
+  for (auto node : loco::output_nodes(g))
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:584]: Can you please add a commnet what this `for` loop does as you explained in the comment?

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 2]
```diff
@@ -581,6 +581,18 @@ bool QuantizeWithMinMaxPass::run(loco::Graph *g)
...
+  {
+    auto circle_node = loco::must_cast<luci::CircleOutput *>(node);
+    if (static_cast<luci::CircleNode *>(circle_node->from())->dtype() == _output_dtype)
+    {
+      circle_node->dtype(_output_dtype);
+      auto graph_outputs = g->outputs();
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:590]: `graph_outputs ` seems to be loop independent

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:584]: Something like this maybe...
1) update output nodes dtype
2) update output dtype

Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:586]: I've added a comment.

**Pull request merged**